src/utils/SubscribableRequestQueue.ts

```diff
@@ -0,0 +1,175 @@
+import RequestQueue from "./RequestQueue";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Resolver = (value?: any) => void;
```

Comment:
could you use unknown instead of any here, to quiet the linter?

Reply:
Tried this, and it seems like `unknown` works for `Rejecter` and not `Resolver` because the argument for `Rejecter` never has a narrower type than `unknown`. But putting `unknown` in `Resolver` causes an error when we try to put the actual resolve function from a promise into a field of type `Resolver`:

```
let resolver: Resolver;
new Promise<T>((resolve) => resolver = resolve); // error: type 'T' is not assignable to type 'unknown'
```

I tried making `Resolver` generic as well, but then we can't store `Resolvers` of lots of different types in one map without using `any`:

`let resolvers: Map<string, Resolver<???>>;`

Either way, `any` does seem to be the only way out here.